### PR TITLE
Migration to Laminas

### PR DESCRIPTION
**Pull request by rarog**
This should replace https://github.com/neilime/zf-twbs-helper-module/pull/27.
- I cleaned the two formatting errors.
- I left out laminas/laminas-dependency-plugin from dependencies, as non of the project dependencies relies on old zend/* dependencies.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
     "name": "neilime/zf-twbs-helper-module",
     "type": "library",
-    "description": "Zend Framework module for easy integration of Twitter Bootstrap",
+    "description": "Zend Framework/Laminas module for easy integration of Twitter Bootstrap",
```
Comment (neilime) [inline composer.json]: Zend Framework/Laminas => Laminas (formerly Zend Framework)

File: composer.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
     "keywords": [
         "Zend Framework",
+        "Laminas",
```
Comment (neilime) [inline composer.json]: Laminas should be the first in the list

File: config/module.config.php [hunk 3]
```diff
@@ -72,7 +72,7 @@
             'formErrors'          => \TwbsHelper\Form\View\Helper\FormErrors::class,
             'formLabel'           => \TwbsHelper\Form\View\Helper\FormLabel::class,
 
-            // ZF3
+            // ZF3 and Laminas
```
Comment (neilime) [inline config/module.config.php]: Laminas and ZF3

File: src/TwbsHelper/Form/View/Helper/Factory/FormCollectionFactory.php [hunk 4]
```diff
@@ -7,26 +7,26 @@
...
+        \Laminas\ServiceManager\ServiceLocatorInterface $oServiceLocator,
         $sCanonicalName = null,
         $sRequestedName = null
     ) {
         return $this($oServiceLocator, $sRequestedName);
     }
 
     /**
-     * Compatibility with ZF3
+     * Compatibility with ZF3 and Laminas
```
Comment (neilime) [inline src/TwbsHelper/Form/View/Helper/Factory/FormCollectionFactory.php]: Laminas and ZF3

File: src/TwbsHelper/Form/View/Helper/Factory/FormElementFactory.php [hunk 5]
```diff
@@ -5,27 +5,27 @@
...
+        \Laminas\ServiceManager\ServiceLocatorInterface $oServiceLocator,
         $sCanonicalName = null,
         $sRequestedName = null
     ) {
         return $this($oServiceLocator, $sRequestedName);
     }
 
     /**
-     * Compatibility with ZF3
+     * Compatibility with ZF3 and Laminas
```
Comment (neilime) [inline src/TwbsHelper/Form/View/Helper/Factory/FormElementFactory.php]: Laminas and ZF3

File: src/TwbsHelper/Form/View/Helper/Factory/FormFactory.php [hunk 6]
```diff
@@ -7,26 +7,26 @@
...
+        \Laminas\ServiceManager\ServiceLocatorInterface $oServiceLocator,
         $sCanonicalName = null,
         $sRequestedName = null
     ) {
         return $this($oServiceLocator, $sRequestedName);
     }
 
     /**
-     * Compatibility with ZF3
+     * Compatibility with ZF3 and Laminas
```
Comment (neilime) [inline src/TwbsHelper/Form/View/Helper/Factory/FormFactory.php]: Laminas and ZF3

File: src/TwbsHelper/Form/View/Helper/Factory/FormRowFactory.php [hunk 7]
```diff
@@ -7,26 +7,26 @@
...
+        \Laminas\ServiceManager\ServiceLocatorInterface $oServiceLocator,
         $sCanonicalName = null,
         $sRequestedName = null
     ) {
         return $this($oServiceLocator, $sRequestedName);
     }
 
     /**
-     * Compatibility with ZF3
+     * Compatibility with ZF3 and Laminas
```
Comment (neilime) [inline src/TwbsHelper/Form/View/Helper/Factory/FormRowFactory.php]: Laminas and ZF3

**Comment by rarog** [comment]
Of course, done!

**Pull request merged**